.github/workflows/main.yml

```diff
@@ -24,10 +24,10 @@ jobs:
       - uses: actions/checkout@v3
       - name: Setup environment
         run: |
-          apk add --no-cache jq
+          apt-get update && apt-get install -y jq
           # secrets are not passed to the runner when a workflow is triggered from a forked repository
           # https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow
-          if [[ -z "${{ secrets.MENDER_SERVER_ACCESS_TOKEN }}" ]]; then
+          if [ -z "${{ secrets.MENDER_SERVER_ACCESS_TOKEN }}" ]; then
```

Comment:
This is not part of the commit message.

Reply:
not sure if such granularity in commit messages fits my common sense understanding, but ok, let's make it even better 🙃

Reply:
I didn't meant to be rude, it just that this came out of the blue in the force-push and I didn't see that it was actually related to the base image change. An small amend to the commit message would have been enough.

In any case I was blocking the PR because my test running the GH workflow failed, not because of this nitpick!